test/tests/functional/pbs_test_tpp.py

```diff
@@ -374,7 +376,7 @@ def test_comm_with_vnode_insertion(self):
             vnode_list = [self.server.status(NODE)[1]['id'],
                           self.server.status(NODE)[2]['id']]
```

Comment:
Only call status() once, and and call ret[0]['id'] and ret[1]['id]], or use list comprehension and say [ret[i]['id'] for i in range(2)] (here and above)